data_acquisition/py/runGEEperTile.py

```diff
@@ -1287,13 +1288,19 @@ def maximum_no_of_tasks(MaxNActive, waitingPeriod):
   poly_feat = ee.FeatureCollection(features)
 
 
-if 'center' in extent:
+if 'polycenter' in extent:
   if yml['polygon'][0] == True:
     centers_csv = read_csv('data_acquisition/out/user_polygon_centers.csv')
+    centers_csv = (centers_csv.rename(columns={'poi_latitude': 'Latitude', 
+      'poi_longitude': 'Longitude',
+      'r_id': 'id'}))
     # load the shapefile into a Fiona object
-    centers = csv_to_eeFeat(centers_csv, yml['poly_crs'][0])
+    centers = csv_to_eeFeat(centers_csv, 'EPSG:4326')
```

Comment:
Had to make some upstream changes for the UTM calculation in calc_centers for compatibility with some CRS's